ITC205_BaseLine_Code/src/PayFineControl.java

```diff
@@ -1,31 +1,30 @@
 public class PayFineControl {
 	
 	private PayFineUI ui;
-	private enum CONTROL_STATE { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
-	private CONTROL_STATE state;
-	
+	private enum ControlState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+	private ControlState state;
 	private library library;
 	private member member;;
 
 
 	public PayFineControl() {
 		this.library = library.INSTANCE();
-		state = CONTROL_STATE.INITIALISED;
+		state = ControlState.INITIALISED;
 	}
 	
 	
 	public void setUI(PayFineUI ui) {
-		if (!state.equals(CONTROL_STATE.INITIALISED)) {
+		if (!state.equals(ControlState.INITIALISED)) {
 			throw new RuntimeException("PayFineControl: cannot call setUI except in INITIALISED state");

```

Comment:
D3 - Put a line break in after PayFineControl:

**this is because that line exceeds the 100 character limit

Reply:
Fixed